app/serializers/api/v2/serializable_course.rb

```diff
@@ -47,7 +47,11 @@ def enrichment_attribute(name, enrichment_name = name)
 
       has_many :site_statuses
       has_many :sites
-      has_many :subjects
+      has_many :subjects do
+        data do
+          @object.course_subjects.sort_by(&:priority).map(&:subject)
+        end
+      end
```

Comment:
I would've expected the `subjects` association to already be ordered correctly, but even better, if we just order `course_subjects` as I suggest above we're definitely covered and don't need to do this.

I tested this locally, btw, and it appears to work.